Cargo.toml

```diff
@@ -35,4 +36,4 @@ anyhow = "1.0.31"
 textwrap = "0.12.1"
 
 [patch.crates-io]
-probe-rs = { git = "https://github.com/probe-rs/probe-rs" }
+probe-rs = { version = "0.8.0", git = "https://github.com/probe-rs/probe-rs" }
```

Comment:
This is not required anymore since it's in the regular dependencies now.

Reply:
I kept it because of the `probe-rs-rtt` dependency, otherwise there are two versions of `probe-rs` used.

Reply:
Uhm probe-rs-rtt cannot use a different version? Otherwise it cannot interop which it does? I don't understand ...

Reply:
Without the patch here, probe-rs-rtt uses the released version, but cargo-embed uses the git version. This then leads to a conflict. With the patch, both use the git version.

Reply:
Ah sorry, confused me for a bit :) Thanks for elaborating!